res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -2227,6 +2469,34 @@ NumarkMixtrack3.OnPFLStatusChange = function(value, group, control) {
     deck.LEDs.headphones.onOff((value) ? ON : OFF);
 };
 
+
+NumarkMixtrack3.OnLoadSelectedTrack = function(value, group, control) {
+    var decknum = parseInt(group.substring(8,9));
+    var deck = NumarkMixtrack3.decks["D" + decknum];
+        
+    var groupOff;
+    var deckOff;
+    if (decknum === 1) {
+        groupOff = "[Channel2]";
+        deckOff = NumarkMixtrack3.decks["D2"];
+    }else{
+        groupOff = "[Channel1]";
+        deckOff = NumarkMixtrack3.decks["D1"];
+    }
+    
+    printInfo ("**** callback OnLoadSelectedTrack");
+    
+    if (smartPFL){
+    printInfo("smartPFL" +smartPFL);
+    printInfo("group = "+group );
+    printInfo("groupOff="+groupOff);
+            engine.setValue(group, 'pfl', true);
+            engine.setValue(groupOff, 'pfl', false);
+            deck.LEDs["headphones"].onOff(ON);
+            deckOff.LEDs["headphones"].onOff(ON);
```

Comment:
The headphones LED is set on one channel. When you load on the other deck, the LED switches to the opposite state with that code. I tested physically on the controller, and LED behavior is good.